kubectl-cloudflow/docker/inspect.go

```diff
@@ -0,0 +1,196 @@
+// This code part is copied and modified from the skopeo project,
+// https://github.com/containers/skopeo.
```

Comment:
Do you need to include some copyright header?

Reply:
Not sure I thought a bit before but will double-check.